### ISSUE TITLE
Error: Dynamic require of "path" is not supported

### ISSUE THREAD

**Issue by yzh990918** [wheatjs/vite-plugin-vue-type-imports#25]
<img width="1266" alt="image" src="https://user-images.githubusercontent.com/46418596/186829370-97afc971-7fcf-417e-aa9d-dd3ae434ec7e.png">

**Comment by yzh990918** [comment]
<img width="624" alt="image" src="https://user-images.githubusercontent.com/46418596/186829617-11673067-c691-4abb-b120-bacd0d0ea83a.png">

**Comment by yzh990918** [comment]
<img width="444" alt="image" src="https://user-images.githubusercontent.com/46418596/186829648-ec49bce2-a301-4145-8d59-66d38803e039.png">

**Comment by Zolyn** [comment]
Can you give a reproduction repo?

**Comment by misogihagi** [comment]
Hi I saw the same issue.
here is my [repo](https://github.com/misogihagi/vitest-test)

easy two step can reproduce.

1. type commands
`npm create vite@latest
`

> Ok to proceed? (y) y
> ✔ Project name: … vite-project
> ✔ Select a framework: › vue
> ✔ Select a variant: › vue-ts
> 

```
cd vite-project/
npm install
npm i vite-plugin-vue-type-imports
```

2. rewrite three files

src/components/props.ts
`export type props={ msg: string }
`

src/components/HelloWorld.vue
```
~~~~~~~~~~~~~~~~~
import type {props} from './props'
defineProps<props>()
~~~~~~~~~~~~~~~~~
```

vite.config.ts
```
import { defineConfig } from 'vite'
import Vue from '@vitejs/plugin-vue'
import VueTypeImports from 'vite-plugin-vue-type-imports'

export default defineConfig({
  plugins: [
    Vue(), 
    VueTypeImports(),
  ],
})
```

**Comment by Zolyn** [comment]
@wheatjs Do you have any release plan? It may be related to the version of the bundler. (See https://github.com/wheatjs/vite-plugin-vue-type-imports/issues/24#issuecomment-1221217545)

**Comment by wheatjs** [comment]
> @wheatjs Do you have any release plan? It may be related to the version of the bundler. (See [#24 (comment)](https://github.com/wheatjs/vite-plugin-vue-type-imports/issues/24#issuecomment-1221217545))

Yes, we can do a release tomorrow

**Comment by wheatjs** [comment]
@Zolyn New version has been released https://github.com/wheatjs/vite-plugin-vue-type-imports/releases/tag/v0.2.1

**Comment by Zolyn** [comment]
It should be fixed in [`v0.2.1`](https://github.com/wheatjs/vite-plugin-vue-type-imports/releases/tag/v0.2.1)

**Comment by misogihagi** [comment]
Hello, thank you for your help before.

By the way, I found another problem.
 
I used 
`
export type props=Partial<{ msg: string }>
`
instead of 
`
export type props={ msg: string } 
`
and the example problem occurred again.

> [vite:vue] [@vue/compiler-sfc] type argument passed to defineProps() must be a literal type, or a reference to an interface or literal type.
> 
> /vitest-test/src/components/HelloWorld.vue
> 5  |  
> 6  |  
> 7  |  defineProps<props>()
>    |              ^^^^^
> 8  |  
> 9  |  
> 
> 

Here is my [repp](https://github.com/misogihagi/vitest-test/tree/vue).
